packages/zk-eddsa-ticket-pcd/artifacts-unsafe/eddsaTicket.circom

```diff
@@ -38,6 +38,18 @@ template PartialTicket () {
     signal input isRevoked;
     signal input revealIsRevoked;
 
+    signal input ticketCategory;
```

Comment:
My same comments on the other PCD apply to this one at all levels, so I won't repeat them.